src/includes/getting-started-primer/dotnet.mdx

```diff
@@ -0,0 +1,16 @@
+Our .NET SDK supports all recent versions of the platform, and integrates well with a variety of popular frameworks and packages in the ecosystem. It gives developers helpful hints for where and why an error or performance issue might have occurred.
+
+This <PlatformLink to="/compatibility/">SDK is compatible</PlatformLink> with .NET Standard 2.0 and .NET Framework 4.6.1. For older versions, such as .NET Framework 3.5, please refer to our [legacy SDK](/clients/csharp/).
+
+**Features:**
+
+- Multiple integrations with additional features such as Serilog, log4net, ASP.NET Core, EntityFramework, Xamarin.
+- <PlatformLink to="/enriching-events/breadcrumbs/#automatic-breadcrumbs">Breadcrumbs automatically</PlatformLink> captured
```

Comment:
```suggestion
- <PlatformLink to="/enriching-events/breadcrumbs/#automatic-breadcrumbs">Breadcrumbs automatically</PlatformLink> captured.
```

Reply:
actually, we don't want periods on bullets. I'll fix as I see this is inconsistent in the list.